cypress/e2e/awx/administration/wfApprovalsList.cy.ts

```diff
@@ -0,0 +1,595 @@
+                      cy.wait('@deleteWFA')
+                        .its('response')
+                        .then((response) => {
+                          expect(response?.statusCode).to.eql(204);
+                        });
+                    }
+                  );
+                });
+            });
+          }
+        );
+        cy.deleteAwxWorkflowJobTemplate(workflowJobTemplate, { failOnStatusCode: false });
+      });
+    });
+  });
+
+  describe('Workflow Approvals - Bulk Approve, Bulk Deny, Bulk Delete', () => {
+    it(`can enable concurrent jobs in a WFJT with a WF approval node, launch multiple jobs, bulk approve, then bulk delete from list toolbar`, () => {
+      cy.createAwxWorkflowJobTemplate({
+        name: 'E2E Workflow Approval-BULK APPROVE-' + randomString(4),
+        organization: organization.id,
+        inventory: inventory.id,
+      }).then((wfjt) => {
+        workflowJobTemplate = wfjt;
+        cy.createAwxWorkflowVisualizerJobTemplateNode(workflowJobTemplate, jobTemplate).then(
+          (jtNode) => {
+            jobTemplateNode = jtNode;
+            cy.createAwxWorkflowVisualizerApprovalNode(workflowJobTemplate).then((appNode) => {
+              approvalWFNode = appNode;
+              cy.createWorkflowJTAlwaysNodeLink(approvalWFNode, jobTemplateNode);
+
+              editWorkflowJobTemplate();
+              workflowApprovalBulkAction('approve');
+              deleteApprovalFromListToolbar();
+            });
+          }
+        );
+        cy.deleteAwxWorkflowJobTemplate(workflowJobTemplate, { failOnStatusCode: false });
+      });
+    });
+
+    it('can enable concurrent jobs in a WFJT with a WF approval node, launch multiple jobs, bulk deny, then bulk delete from list toolbar', () => {
+      cy.createAwxWorkflowJobTemplate({
+        name: 'E2E Workflow Approval-BULK DENY-' + randomString(4),
+        organization: organization.id,
+        inventory: inventory.id,
+      }).then((wfjt) => {
+        workflowJobTemplate = wfjt;
+        cy.createAwxWorkflowVisualizerJobTemplateNode(workflowJobTemplate, jobTemplate).then(
+          (jtNode) => {
+            jobTemplateNode = jtNode;
+            cy.createAwxWorkflowVisualizerApprovalNode(workflowJobTemplate).then((appNode) => {
+              approvalWFNode = appNode;
+              cy.createWorkflowJTAlwaysNodeLink(approvalWFNode, jobTemplateNode);
+
+              editWorkflowJobTemplate();
+              workflowApprovalBulkAction('deny');
+              deleteApprovalFromListToolbar();
+            });
+          }
+        );
+        cy.deleteAwxWorkflowJobTemplate(workflowJobTemplate, { failOnStatusCode: false });
+      });
+    });
+  });
+
+  /* 
+  Used in the Workflow Approvals - Bulk Approve, Bulk Deny, Bulk Delete tests (below)
+  **/
+  function editWorkflowJobTemplate() {
+    cy.visit(`/templates/workflow-job-template/${workflowJobTemplate.id.toString()}/edit`);
```

Comment:
We should not be using `cy.visit` and we should not be hard coding url paths in the the tests